### PR TITLE
created click button to congratulate/animate

### PR DESCRIPTION
**Pull request by mariarabin**
Pls. review. Thanks!

### CODE REVIEW

File: congratulate.html [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+                //var celebs = [];
+                //var randomCeleb = getRandomArrayElement(celebs);
+
+                $("#gifid").html(`<img src="${celeb}" style="margin-top:10px"; height = '900' width = '900' />`);
+            });
+        };
+
+        function playSound() {
+            var audioElement = document.createElement("audio");
+            audioElement.setAttribute("src", "./js/clap-mp3.mp3");
```
Comment (alexjzyang) [inline congratulate.html:54]: the sound is also hard coded

File: congratulate.html [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+            $(".button").hide();
+            celebrate();
+            playSound();
+        }
+
+
+        // Function to run when the button is clicked
+        function celebrate() {
+
+            var queryURL = "https://api.tenor.com/v1/search?q=minions&key=B3RT5ZFYTZ7W&limit=20";
```
Comment (alexjzyang) [inline congratulate.html:35]: topic search term is hard coded, so it plays the same animation all the time.

**Review by alexjzyang — CHANGES_REQUESTED** [review]
Maria, this is great work. The ajax works very well. It's just they are static, referencing the same topic. And I believe you want to do another API call for the sound right? currently it's static as well.

Let me know if you want me to pull this anyway to master, and go from there.

File: congratulate.html (hunk 1 above)
Comment (mariarabin) [reply congratulate.html:54]: no changes made

**Review by alexjzyang — APPROVED** [review]

**Pull request merged**